auslib/web/admin/views/permissions.py

```diff
@@ -144,23 +143,36 @@ def _delete(self, username, permission, changed_by, transaction):
 
 
 class PermissionScheduledChangesView(ScheduledChangesView):
+    """/scheduled_changes/permissions"""
+
     def __init__(self):
         super(PermissionScheduledChangesView, self).__init__("permissions", dbo.permissions)
 
     @requirelogin
     def _post(self, transaction, changed_by):
         change_type = connexion.request.json.get("change_type")
+        connexion.request.json.pop("csrf_token", None)
 
-        if change_type == "update":
-            form = ScheduledChangeExistingPermissionForm()
-        elif change_type == "insert":
-            form = ScheduledChangeNewPermissionForm()
-        elif change_type == "delete":
-            form = ScheduledChangeDeletePermissionForm()
-        else:
-            return Response(status=400, response="Invalid or missing change_type")
+        what = {}
+        for field in connexion.request.json:
+            if change_type == "delete" and field == "options" or change_type == "insert" and field == "data_version":
```

Comment:
not via the UI forms AFAIK. However, any client can still pass it accidentally and the newly created rule can end up with a non-single (not 1) value. Hence the check